app/data_model/database.py

```diff
@@ -60,6 +60,19 @@ def __repr__(self):
         return "<EQSession('%s', '%s', '%s')>" % (self.eq_session_id, self.user_id, self.timestamp)
 
 
+class UsedToken(base):
+    __tablename__ = "used_token"
+    token = Column("token", String, primary_key=True)
+    timestamp = Column("timestamp", DateTime)
```

Comment:
what's this a timestamp of 'created/seen/used' ? consider naming to reflect usage.